apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -506,6 +507,8 @@ namespace MWRender
         mRootNode->getOrCreateStateSet()->setAttribute(Shader::RemovedAlphaFunc::getInstance(GL_ALWAYS));
         // The transparent renderbin sets alpha testing on because that was faster on old GPUs. It's now slower and breaks things.
         mRootNode->getOrCreateStateSet()->setMode(GL_ALPHA_TEST, osg::StateAttribute::OFF);
```

Comment:
Explicitly disabling `GL_ALPHA_TEST` was only done to fix the issue that this new change also fixes, so that line could be removed as it's now redundant.